### ISSUE TITLE
Plan ran into dataextensionmethodcallexception

### ISSUE THREAD

**Issue by CrispyDairy** [plan-player-analytics/Plan#2051]
### Describe the issue

[DataExtensionMethodCallException-785ebed7b5.txt](https://github.com/plan-player-analytics/Plan/files/6999850/DataExtensionMethodCallException-785ebed7b5.txt)

![image](https://user-images.githubusercontent.com/81482501/129724435-4f69f069-5fc6-42ae-8a69-f8d41ec70819.png)

### Plugin versions

Version: 5.4 build 1366

### Additional information

Just popped out of nowhere.

**Comment by AuroraLS3** [comment]
I guess they had to change what they use to represent claim corners and had to remove the method Plan is using

**Comment by CrispyDairy** [comment]
Anything I need to do?

**Comment by AuroraLS3** [comment]
Not really, it just doesn't gather that particular table until this gets fixed

**Comment by poqdavid** [comment]
The problem is happening again in version `5.4 build 1633`
[DataExtensionMethodCallException-ffa4d867bd.txt](https://github.com/plan-player-analytics/Plan/files/8599079/DataExtensionMethodCallException-ffa4d867bd.txt)

**Forge**: 14.23.5.2860
**Sponge**: 2838-7.4.3
**GriefDefender**: 2.0.1
